src/main/resources/templates/community.html

```diff
@@ -30,31 +30,46 @@
 				</div>
 			</div>
 
-			<section id="joinUs" class="join-us">
 				<div class="container ">
 					<div class="row ">
 					  <div class="heading col-sm-10 col-sm-offset-1 ">
 						<div class="col-sm-6">
-							<h20><strong>Research&amp;Development</strong> </h20>
+							<h20>Research&amp;Development </h20>
 						</div>
 						<div class="col-sm-6">
 						  <p align="justify">With 1-click ready-to-use environments and necessary resources, researchers can focus on research core activities such as innovation. </p>
 						  <p align="justify"> The lab infrastructure is capable of supporting large scale experimentation, validation and demonstration.</p>
 						</div>
 					  </div>
 						 <div class="heading  col-sm-10 col-sm-offset-1 ">
-							 <div class="col-sm-6"> <h20><strong>Education&amp;Training</strong></h20>
+							 <div class="col-sm-6"> <h20>Education&amp;Training</h20>
 							 </div>
 						     <div class="col-sm-6">
 		<p align="justify">A large collection teaching materials are available to demonstrate realistic environments in teaching. Security environments are customizable to suit teaching needs.The lab infrastructure is also capable of supporting large scale cyber exercises.</p>
 		</div>
 		</div>
 		</div> <!-- row -->
+					<div class="cc-wraper">
+						<div class="cc-contactpop fixed">
+							<h1>Do you need Help? <span>Get in touch.</span></h1>
+							<div class="renseignement">
+								<i class="fa fa-times"></i>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)